source/ui/dataman/AMScanView.h

```diff
@@ -323,26 +346,17 @@ class AMScanViewMultiView : public AMScanViewInternal {
 
 public:
 	explicit AMScanViewMultiView(AMScanView* masterView);
-
 	virtual ~AMScanViewMultiView();
 
 public slots:
-
-	/// Sets the visibility of the plot item cursor.
-	void setPlotCursorVisibility(bool visible);
-	/// Sets the coordinates for the plot item cursor.
-	void setPlotCursorCoordinates(const QPointF &coordinates);
-	/// Overloaded.  Sets the x-axis coordinate only.
-	void setPlotCursorCoordinates(double xCoordinate);
-	/// Sets the color of the plot cursor.
-	void setPlotCursorColor(const QColor &color);
-
```

Comment:
During compile I get errors related to classes which still call these now missing functions. Are these now handled internally, or is there some other mechanism that external classes can change these settings?

An example is REIXSRIXSScanConfigurationView::addScan() and ::onEnergySpinBoxChanged().